src/components/Layout/SymptomsCarousel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import SymptomsCards from '../SymptomsCards'
+import './SymptomsCarousel.scss';
+import '../../../styles/_colors.scss';
+import Tiredness from '../../../assets/images/tiredness.svg';
+import Cough from '../../../assets/images/cough.svg';
+import Fever from '../../../assets/images/fever.svg';
+import Diarrhoea from '../../../assets/images/diarrhoea.svg';
+import RunnyNose from '../../../assets/images/runny-nose.svg';
+import ShortnessBreath from '../../../assets/images/shortness-breath.svg';
+
+
```

Comment:
I think you can create an array with all the information of the symptoms, something like this:
```js
const symptoms = [
    {
        title: 'Tiredness',
        description: 'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam',
        image: Tiredness
    },
    {
        title: 'Fever',
        description: 'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam',
        image: Fever
    }
]
```

With this, you can use `symptoms.map()` to render the card.